cl/users/templates/profile/donations.html

```diff
@@ -19,115 +19,29 @@
-                     data-enable-icon-class="fa-play"
-                     data-disable-icon-class="fa-pause"></i> <span class="alert_btn_txt">
-                  {% if donation.enabled %}
-                    Pause Contributions
-                  {% else %}
-                    Curerntly Disabled &mdash; Re-Enable Contributions?
-                  {% endif %}</span>
-                </a>
-              </td>
-            </tr>
-          {% endfor %}
-          </tbody>
-        </table>
-        <h3 class="caps gray">
-          Questions about monthly donations? <a href="{% url "donation_help" %}"
-             class="btn btn-default btn-lg">Read the FAQs</a>
-        </h3>
-      </div>
-    {% endif %}
-
-    {% if user.donations.all and user.profile.email_confirmed %}
-      <hr class="v-offset-above-3">
-      <div class="row">
-        <div class="hidden-xs col-sm-1 col-md-3"></div>
-        <div class="col-xs-12 col-sm-10 col-md-6 text-center">
-          <h2>Thank You for Your Support <i class="fa fa-heart red"></i></h2>
-          <p>Your support and contributions make a huge difference. If you would like to change a donation or if you have any questions, please get in touch with us at <a href="mailto:donate@free.law">donate@free.law</a>. We'd be happy to hear from you.
-          </p>
-        </div>
-        <div class="hidden-xs col-sm-1 col-md-3"></div>
-      </div>
-    {% endif %}
-
-    {% if not user.donations.all and not user.monthly_donations.all %}
-      <div class="row">
-        <div class="hidden-xs col-sm-1 col-md-3"></div>
-        <div class="col-xs-12 col-sm-10 col-md-6 text-center">
-          <h1>Donate to Support Free Law Project</h1>
-          <h3 class="caps gray">We need your support!</h3>
-          <p class="v-offset-above-2">CourtListener and RECAP are initiatives of <a href="https://free.law">Free Law Project</a> and are supported by user donations. Our biggest expenses are our full time developers and our server&nbsp;costs.
-          </p>
-          <p class="v-offset-above-2"><a href="{% url "donate" %}?referrer=profile"
-                class="btn btn-lg btn-danger">Donate Now</a>
-          </p>
-          {% if not user.profile.email_confirmed %}
-            <p class="gray small">(If you have already donated, you may need to confirm your email address to see your past donations.)
-            </p>
-          {% endif %}
-        </div>
-        <div class="hidden-xs col-sm-1 col-md-3"></div>
-      </div>
-    {% endif %}
+  <div class="flex flex-row justify-content-center col-xs-12">
+    <div class="col-xs-12 col-sm-10 col-md-6">
+      {% if user.profile.is_member %}
+      <h1 class="text-center v-offset-above-2">
+        Thank you for being a member of Free Law Project! <i class="fa fa-heart red"></i>
+      </h1>
+      <p class="v-offset-above-2">
+        Your ongoing membership fuels our passion to keep developing these and other features that makes the judiciary more equitable.
```

Comment:
"These and other features" is plural, so:

```suggestion
        Your ongoing membership fuels our passion to keep developing these and other features that make the judiciary more equitable.
```

Reply:
oh sorry. Thank you for bringing it to my attention.